app/api/dao/mentorship_relation.py

```diff
@@ -205,3 +205,31 @@ def cancel_relation(user_id, relation_id):
         request.save_to_db()
 
         return {'message': 'Mentorship relation was cancelled successfully.'}, 200
+
+    @staticmethod
+    def delete_request(user_id, request_id):
+
+        user = UserModel.find_by_id(user_id)
+
+        # verify if user exists
+        if user is None:
+            return {'message': 'User does not exist.'}, 404
+
+        request = MentorshipRelationModel.find_by_id(request_id)
+
+        # verify if request exists
```

Comment:
it is a good practice to avoid comments that include "what" a code line is doing rather that "why" a code line is there.

Reply:
Thank you @Dilu9218! In the future for this case should I avoid these comments at all or does this code need a "why"?